examples/with-mocha/src/Child.test.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import { render, screen, waitFor, cleanup } from '@testing-library/react'
+import { rest } from 'msw'
+import Child from './Child'
+import mswServer from '../mocks/server'
```

Comment:
Do you think we could name it just `server`? I don't mind it being name differently, but we should stick to the same name throughout examples, which I think is `server`.